doc/monitoring-stack-deployment/grafana-v5.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: grafana.integreatly.org/v1beta1
+kind: Grafana
+metadata:
+  name: example-grafana
+  labels:
+    apim-management: grafana
+spec:
+  config:
+    log:
+      mode: "console"
+      level: "debug"
+    security:
+      admin_password: "1234"
+      admin_user: "admin"
+    auth:
+      disable_login_form: "false"
+      disable_signout_menu: "true"
+    auth.basic:
+      enabled: "true"
+    auth.anonymous:
+      enabled: "true"
+  dashboardLabelSelector:
```

Comment:
@MStokluska , please see [here](https://grafana.github.io/grafana-operator/blog/2023/05/27/v4-to-v5-migration/) -- "The biggest difference to keep in mind is that the label selector isn’t in a Grafana spec anymore (dashboardLabelSelector), it’s in other CRs instead (instanceSelector)"  .

Reply:
Thanks, other resources rely on label on Grafana CR that is a requirement for v5. I think the dashboardLabelSelector could be removed.